test/functional/mempool_accept.py

```diff
@@ -104,7 +105,7 @@ def run_test(self):
         ))['hex']
         tx.deserialize(BytesIO(hex_str_to_bytes(raw_tx_final)))
         self.check_mempool_result(
-            result_expected=[{'txid': tx.rehash(), 'allowed': True}],
+            result_expected=[{'txid': tx.rehash(), 'allowed': True, 'vsize': 188, 'fees': { 'base': Decimal(str(int(coin['amount']) - 0.025))}}],
```

Comment:
Can we also try to make these fee calculations more programmatic? Otherwise it's a burden on future test writers.